src/components/input/Input.vue

```diff
@@ -218,6 +240,47 @@ export default {
             this.$nextTick(() => {
                 this.$refs.input.focus()
             })
+        },
+		
+		shouldUpdateLocalValue (localValue, checkVal, fromBlur = false) {
+            return this.isNotInFocusAndDirty(localValue, checkVal, fromBlur)||this.isDirtyWithModifiers(localValue, checkVal, fromBlur);
+        },
+		
+        isNotInFocusAndDirty (localValue, checkVal, fromBlur) {
+            if(!fromBlur){//from value (v-model) watcher
+                if(this.isFocused && !this.forcedSync){
+                    if(this.trim||this.number)return false;//don't update value if it's trim/number and it's still focused
+                }
+                return true;//update value if it's trim/number and unfocused
+            }
+            //sync local variable on blur, default v-model behaviour
+            return localValue !== checkVal;
```

Comment:
If it's gonna be called from localOnBlur and values not same we gonna force update them.